bedrock/firefox/templates/firefox/privacy/products.html

```diff
@@ -0,0 +1,187 @@
+  {{ call_out(
+    title=_('Firefox <strong>protects</strong> your privacy in every product')|safe,
+    class='privacy-products-call-out-main',
+    heading_level=1)
+  }}
+
+  {% call hero(
+    title='Firefox Browser',
+    class='privacy-products-hero mzp-has-image mzp-t-dark t-medium',
+    image_url='firefox/privacy/products/report.svg',
+    include_cta=True,
+    heading_level=2
+  ) %}
+    <h3 class="privacy-products-hero-sub-title">{{ _('2,000+ types of trackers blocked — automatically') }}</h3>
+
+    <p>
+    {% trans %}
+      Tracking has become an epidemic online: companies follow every move,
+      click and purchase, collecting data to predict and influence what
+      you’ll do next. We think that’s a gross invasion of your privacy.
+      That’s why Firefox mobile and desktop browsers have Enhanced Tracking
+      Protection on by default.
+    {% endtrans %}
+    </p>
+
+    <p class="show-mobile is-js-dependent">
+      <strong>{{ _('If you want to see what Firefox is blocking for you, visit this page on your Firefox desktop browser.') }}</strong>
+    </p>
+
+    <p class="show-firefox-desktop-70 is-js-dependent">
+      <a class="mzp-c-cta-link js-open-about-protections" href="https://support.mozilla.org/en-US/kb/enhanced-tracking-protection-firefox-desktop/?utm_source={{ _entrypoint }}&utm_medium=referral&utm_campaign={{ _utm_campaign }}&entrypoint={{ _entrypoint }}">
+        {{ _('See what Firefox has blocked for you') }}
+      </a>
+    </p>
+
+    {# L10n: "Enhanced Tracking Protection" is a feature name; it should be capitalized. #}
+    <p class="privacy-products-hero-tagline show-default show-firefox-desktop-old">{{ _('Get Enhanced Tracking Protection') }}</p>
+
+    <div class="show-default">
+      {{ download_firefox(dom_id='download-button-primary-non-firefox', alt_copy=_('Download the Firefox browser'), download_location='primary cta') }}
+    </div>
+
+    <p class="show-firefox-desktop-old is-js-dependent">
+      <a class="mzp-c-cta-link" href="https://support.mozilla.org/kb/update-firefox-latest-release/?utm_source={{ _entrypoint }}&utm_medium=referral&utm_campaign={{ _utm_campaign }}&entrypoint={{ _entrypoint }}">
+        {{_ ('Update your Firefox browser') }}
+      </a>
+    </p>
+  {% endcall %}
+
+  <div class="privacy-products-etp">
+    <ul class="mzp-l-content mzp-l-card-third mzp-t-dark">
+      {{ picto_card(title=_('Invisible to the top trackers'), desc=_('Meet four of the most common categories of trackers — who won’t meet you.'), class='invisible js-open-about-protections') }}
+      {{ picto_card(title=_('Always in your control'), desc=_('Want to customize what gets blocked? Your settings are only one click away.'), class='control js-open-about-protections') }}
+      {{ picto_card(title=_('Protection beyond tracking'), desc=_('If you have a Firefox account, you can also see how we’re helping you protect your personal info and passwords.'), class='protection js-open-about-protections') }}
```

Comment:
This is a last minute request from the main stakeholder, so it needs to stay. They feel these headings are more eye catching than the CTA links (which are on the small side), 

I agree on the semantics though, so I'll see if we can make just the headings clickable instead of the combined copy.